packages/inter-protocol/src/psm/psmCharter.js

```diff
@@ -46,7 +49,9 @@ export const start = async (zcf, privateArgs) => {
     );
   };
 
-  const makeOfferFilterInvitation = () => {
+  /** @param {Instance} instance */
```

Comment:
@turadg I'm interested to get some training on using `Instance` types parameterized by the contract `start` function and such.

Reply:
check out https://github.com/Agoric/agoric-sdk/compare/type-instance